code/game/gamemodes/endgame/supermatter_cascade/universe.dm

```diff
@@ -87,6 +92,16 @@
 			APC.emagged = 1
 			APC.queue_icon_update()
 
+			APC.areaMaster.power_light = 0
+			APC.areaMaster.power_equip = 0
+			APC.areaMaster.power_environ = 0
+			APC.areaMaster.power_change()
```

Comment:
I'll remove it from hell_universe as well. I originally wrote that before you pushed your fix.